light-client/src/lib.rs

```diff
@@ -23,15 +23,18 @@ pub mod errors;
 pub mod evidence;
 pub mod fork_detector;
 pub mod light_client;
-pub mod operations;
 pub mod peer_list;
-pub mod predicates;
 pub mod state;
 pub mod store;
 pub mod supervisor;
-pub mod types;
 
 pub(crate) mod utils;
 
+// Re-export `tendermint-light-client-verifier` crate.
+pub use tendermint_light_client_verifier as verifier;
+
+// Re-export for backward compatibility
+pub use verifier::{operations, predicates, types};
```

Comment:
I think this is better suited to the v0.23.x series. Since we're dealing with an explicitly breaking change here (tendermint-rs v0.24.0) we may as well abandon the backward compatibility. Especially given that it's basically just ibc-rs that uses this crate at present as far as I can see. See https://crates.io/crates/tendermint-light-client/reverse_dependencies

Reply:
In other words, I don't think we need https://github.com/informalsystems/tendermint-rs/pull/1071/commits/a4d7dc76831e312d07d42d1d37f2c936e8c8ac2a for `master` at this point. What do you think? Do you think it's necessary for ibc-rs for us to maintain this backward compatibility?

Reply:
Actually don't worry about it - we can remove these in a subsequent PR. I think we're good to go here.